lib/pubsub/subscription.js

```diff
@@ -267,12 +270,13 @@ function Subscription(pubsub, options) {
 
   this.autoAck = is.boolean(options.autoAck) ? options.autoAck : false;
   this.closed = true;
-  this.interval = is.number(options.interval) ? options.interval : 10;
   this.inProgressAckIds = {};
+  this.interval = is.number(options.interval) ? options.interval : 10;
   this.maxInProgress =
     is.number(options.maxInProgress) ? options.maxInProgress : Infinity;
   this.messageListeners = 0;
   this.paused = false;
+  this.timeout = options.timeout;
```

Comment:
I looked at the request code, and it looks like it'll skip right by it: https://github.com/request/request/blob/c4d3c9c3058bf1b5a85039c6e428ad6de99bfe1b/request.js#L752